dcrec/secp256k1/schnorr/ecdsa.go

```diff
@@ -161,11 +161,13 @@ func schnorrSign(msg []byte, ps []byte, k []byte,
 // nonceRFC6979 is a local instantiation of deterministic nonce generation
 // by the standards of RFC6979.
 func nonceRFC6979(privKey []byte, hash []byte, extra []byte, version []byte, extraIterations uint32) []byte {
-	bigK := secp256k1.NonceRFC6979(privKey, hash, extra, version,
-		extraIterations)
+	k := secp256k1.NonceRFC6979(privKey, hash, extra, version, extraIterations)
+	kBytes := k.Bytes()
+	defer zeroArray(&kBytes)
+	bigK := new(big.Int).SetBytes(kBytes[:])
 	defer bigK.SetInt64(0)
```

Comment:
Ok not to use `zeroBigInt` here too?

Reply:
It's existing code that I didn't update, but yes it really should be updated as well.  I haven't got around to overhauling the `schnorr` package yet, but that's next up and a large reason for all of this `secp256k1` work to begin with.  I'll open a separate PR to address this in the `schnorr` package since there are more cases as well.